.circleci/config.yml

```diff
@@ -188,6 +227,14 @@ workflows:
               only: /.*/
             tags:
               only: /.*/
+      - coverage:
+          requires:
+            - linux-debian
+          filters:
+            branches:
+              only: /.*/
+            tags:
+              only: /.*/ 
```

Comment:
How long is it taking, should we run it only on nightly?

Reply:
I don't think it should take very long after I fix the mixup with the Valgrind build. It should be possible to run it per-commit.

Reply:
:+1: